docs/topics_5/rn-new-features.adoc

```diff
@@ -5,13 +5,62 @@
 
 This section describes the new features of the {ProductName} {DocInfoProductNumber}.
 
-Updated {WebName} interface::
+== Updated {WebName} interface
 The {WebName} has been redesigned with PatternFly 4. In addition to its general improved appearance, the {WebName} adds elements for project creation and management and updated transformation target icons.
```

Comment:
Try to reword this so that the elements and icons have a parallel grammatical structure.